src/main/java/Elon/Storage.java

```diff
@@ -0,0 +1,86 @@
+package Elon;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
+public class Storage {
+    private final String path;
+
+    private File file;
+
+    public Storage(String path) throws IOException {
+        this.path = path;
+        this.file = new File(this.path);
+        if (!file.exists()) {
+            if (file.createNewFile()) {
+                System.out.println("File created");
+            } else {
+                System.out.println("Failed to create file");
+            }
+        } else {
+            System.out.println("File exists");
+        }
+    }
+
+    private Task parseTask(String line) throws ElonException {
+        String[] elements = line.split(" \\| ");
+        String type = elements[0];
+        boolean isDone = elements[1].equals("1");
+        String description = elements[2];
+
+        switch (type) {
+            case "T":
+                return new ToDo(description, isDone);
+            case "D":
+                LocalDate by = LocalDate.parse(elements[3]);
+                return new Deadline(description, isDone, by);
+            case "E":
+                LocalDate start = LocalDate.parse(elements[3]);
+                LocalDate end = LocalDate.parse(elements[4]);
+                return new Event(description, isDone, start, end);
+            default:
+                throw new ElonException("Invalid task format.");
+        }
```

Comment:
Since this is a switch case statement, please avoid indentation for `case`, as per the coding standard:

<img width="987" alt="Screenshot 2024-09-03 at 11 45 02 AM" src="https://github.com/user-attachments/assets/92cfeb17-b1ac-4af0-9055-5f89621f27f9">